components/login/provider/index.jsx

```diff
@@ -21,7 +21,7 @@
 
 /* eslint react/jsx-props-no-spreading: 0 */
 
-import React, { createRef, useEffect, useState } from "react";
```

Comment:
I am deeply sorry for how the diff in this file ended up, it's pretty nasty, I know. Try switching to side-by-side.